elektronLogging/logger.go

```diff
@@ -0,0 +1,49 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	. "github.com/spdfg/elektron/elektronLogging/types"
+	"strings"
+	"time"
+)
+
+var config LoggerConfig
+var logger *log.Logger
+var formatter ElektronFormatter
+var ElektronLog *LoggerImpl
```

Comment:
Logger and Log are synonyms and therefore should not be used interchangeably.

Reply:
Changed `ElektronLog` to `ElektronLogger`